src/serac/physics/solid_legacy.hpp

```diff
@@ -581,6 +581,8 @@ class SolidLegacy : public BasePhysics {
    */
   mfem::Vector previous_;
 
+  double ode_time_point_;
```

Comment:
```suggestion
  /// @brief The value of time at which the ODE solver wants to evaluate the residual
  double ode_time_point_;
```

Reply:
I moved this to the `BasePhysics` class and added the doxygen comment.